content/en/docs/installation/operator.md

```diff
@@ -259,7 +155,105 @@ kubectl -n <namespace> describe spinnakerservice spinnaker
 kubectl -n <namespace> delete spinnakerservice spinnaker
 ```
 
-## Migrating from Halyard to Operator
+
+## Managing Configuration
+
+### Kustomize
+Because Spinnaker's configuration is now a Kubernetes manifests, we can manage `SpinnakerService` and related manifests in a consistent and repeatable way with [kustomize](https://kustomize.io/).
+
+An example can be found [here](https://github.com/armory-io/spinnaker-operator/tree/master/deploy/spinnaker/kustomize).
+
+```bash
+kubectl create ns spinnaker
+kustomize build deploy/spinnaker/kustomize | kubectl -n spinnaker apply -f -
+```
+
+There are many more possibilities such as:
+- managing manifests of MySQL instances
+- ensuring the same configuration is used between Staging and Production Spinnaker
+- splitting accounts in their own kustomization for an easy to maintain configuration 
+
+### Secret Management
+
+You can store secrets in one of the [support secret engine](../../spinnaker-install-admin-guides/secrets/secrets/#supported-secret-engines). 
+
+#### Kubernetes Secret
+With the operator, you can also reference secrets stored in existing Kubernetes secrets in the same namespace as Spinnaker.
+
+The format is:
+- `encrypted:k8s!n:<secret name>!k:<secret key>` for string values. These will be added as environment variable to the Spinnaker deployment.
+- `encryptedFile:k8s!n:<secret name>!k:<secret key>` for file references. Files will come from a volume mount in the Spinnaker deployment.
+
+
+#### Custom Halyard Configuration
+
+To override Halyard's configuration, create a `ConfigMap` with the configuration changes you need. For example, if using [secrets management with Vault]({{< ref "secrets-vault" >}}), Halyard and Operator containers will need your Vault configuration:
+
+```yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: halyard-custom-config
+data:
+  halyard-local.yml: |
+    secrets:
+      vault:
+        enabled: true
+        url: <URL of vault server>
+        path: <cluster path>
+        role: <k8s role>
+        authMethod: KUBERNETES
+```
+
+You can then mount it in the operator deployment and make it available to the Halyard and Operator containers:
+
+```yaml
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: spinnaker-operator
+  ...
+spec:
+  template:
+    spec:
+      containers:
+      - name: spinnaker-operator
+        ...
+        volumeMounts:
+        - mountPath: /opt/spinnaker/config/halyard.yml
+          name: halconfig-volume
+          subPath: halyard-local.yml
+      - name: halyard
```

Comment:
Curious, Halyard might need this to be set, i'm not 100% sure tho. I know we have it set in our [prod operator config](https://github.com/armory-io/armory-infrastructure/blob/master/spinnaker/prod-operator/deployment.yaml#L40-L42)

```
          env:
          - name: SPRING_PROFILE_ACTIVE
            value: local
```

Reply:
I don't believe so (I think local is standard but I'm not sure). This part is a carryover from the existing doc and I'll let others continue to improve and correct the doc once it's reorganized.